src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,361 @@
+            "Expected, that user with invalid login call yhe RegistrationException";
+    private static final String EXPECTED_MESSAGE_NULL_AGE_USER =
+            "Expected, that user with null age call yhe RegistrationException";
+    private static final String EXPECTED_MESSAGE_INVALID_AGE_USER =
+            "Expected, that user with invalid login call yhe RegistrationException";
+    private static final String EXPECTED_MESSAGE_NULL_PASSWORD_USER =
+            "Expected, that user with null password call yhe RegistrationException";
+    private static final String EXPECTED_MESSAGE_INVALID_PASSWORD_USER =
+            "Expected, that user with invalid password call yhe RegistrationException";
+    private static final int ADDED_FIRST_CUSTOM_USER = 3;
+    private static final int ADDED_SECOND_CUSTOM_USER = 4;
+    private static final int ADDED_THIRD_CUSTOM_USER = 5;
+
+    private static User userBob;
+    private static User userAnn;
+    private static User userRoma;
+    private RegistrationService registrationService;
+
+    @BeforeAll
+    static void setDataBase() {
+        userBob = new User("user_bob", "qwerty1234", 34);
+        userAnn = new User("user_ann", "12345678", 18);
+        userRoma = new User("user_roma", "bobocode", 22);
+        Storage.people.addAll(List.of(userAnn, userBob, userRoma));
+    }
+
+    @BeforeEach
+    void setRegistrationService() {
+        registrationService = new RegistrationServiceImpl();
+    }
+
+    @AfterEach
+    void afterEach() {
+        if (Storage.people.size() > STANDARD_DB_SET_COUNTER) {
+            Storage.people.subList(STANDARD_DB_SET_COUNTER, Storage.people.size()).clear();
+        }
+    }
+
+    @Test
+    @Order(1)
+    void registrationUser_nullUser_notOk() {
+        User nullUser = null;
+        assertThrows(RegistrationException.class,
+                () -> registrationService.register(nullUser),
+                EXPECTED_MESSAGE_NULL_USER);
+        assertThrows(RegistrationException.class,
+                () -> registrationService.register(null),
+                EXPECTED_MESSAGE_NULL_USER);
+    }
+
+    @Test
+    @Order(2)
+    void registrationUser_nullLogin_notOk() {
+        User userNullLoginFirst = new User(null, "fgjhdn1356nb7", 32);
+        User userNullLoginSecond = new User(null, "234vt7d124v1", 28);
+        User userNullLoginThird = new User(null, "adsfrt4v41", 18);
+        assertThrows(RegistrationException.class,
+                () -> registrationService.register(userNullLoginFirst),
+                EXPECTED_MESSAGE_NULL_LOGIN_USER);
+        assertThrows(RegistrationException.class,
+                () -> registrationService.register(userNullLoginSecond),
+                EXPECTED_MESSAGE_NULL_LOGIN_USER);
+        assertThrows(RegistrationException.class,
+                () -> registrationService.register(userNullLoginThird),
+                EXPECTED_MESSAGE_NULL_LOGIN_USER);
```

Comment:
Please let's remove redundant code and simplify it. You don't really need to test 3 similar users with null login here. Fix the same issues everywhere